contracts/cairo/deploy.sh

```diff
@@ -33,7 +27,7 @@ echo -e "- ${PINK}[ETH] Ethereum ContractAddress: $ETH_CONTRACT_ADDR${COLOR_RESE
 echo -e "- ${PINK}[ETH] Market Maker: $MM_ETHEREUM_WALLET${COLOR_RESET}"
 
 echo -e "${GREEN}\n=> [SN] Deploy Escrow${COLOR_RESET}"
-ESCROW_CONTRACT_ADDRESS=$(starkli deploy --watch --rpc $SN_RPC_URL $ESCROW_CLASS_HASH \
+ESCROW_CONTRACT_ADDRESS=$(starkli deploy --watch --rpc $STARKNET_RPC $ESCROW_CLASS_HASH \
```

Comment:
This deploys the Escrow, but i can't seem to find if the proxy how or where the proxy is deployed?

Reply:
From what I understood from the OpenZeppelin documentation, it's not actually a proxy. Instead, the upgradeable concept can be applied to a contract natively, eliminating the need for a proxy. Source: https://docs.openzeppelin.com/contracts-cairo/0.8.0/upgrades